query/src/test/java/org/infinispan/query/api/AnotherTestEntity.java

```diff
@@ -11,17 +11,17 @@
 @Indexed(index="indexB")
 public class AnotherTestEntity implements Serializable {
 
-   private final String value;
+   private final String name;
```

Comment:
Why the renaming?

Reply:
Actually it is not necessary to rename the field, but it is necessary to align the property name and the index field name. Because we still have [this open issue](https://hibernate.atlassian.net/browse/HSEARCH-3589) on Search and without providing the index metamodel the Ickle parsing can use only the Pojo property model to extract the fields :(
As soon as the issue is solved, we will be able to remove this very serious limitation.